line-sdk/src/main/java/com/linecorp/linesdk/openchat/ui/ProfileInfoFragment.kt

```diff
@@ -0,0 +1,91 @@
+package com.linecorp.linesdk.openchat.ui
+
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import androidx.databinding.DataBindingUtil
+import androidx.fragment.app.Fragment
+import androidx.lifecycle.Observer
+import androidx.lifecycle.ViewModelProviders
+
+import com.linecorp.linesdk.R
+import com.linecorp.linesdk.databinding.ProfileInfoFragmentBinding
+import com.linecorp.linesdk.openchat.TextUpdateWatcher
+import kotlinx.android.synthetic.main.profile_info_fragment.displayNameEditText
+import kotlinx.android.synthetic.main.activity_create_open_chat.toolbar
+
+class ProfileInfoFragment : Fragment() {
+
+    private lateinit var binding: ProfileInfoFragmentBinding
+
+    companion object {
+        fun newInstance() = ProfileInfoFragment()
+    }
+
+    private lateinit var viewModel: OpenChatInfoViewModel
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        setHasOptionsMenu(true)
+    }
+
+    override fun onCreateView(
+        inflater: LayoutInflater, container: ViewGroup?,
+        savedInstanceState: Bundle?
+    ): View? {
+        binding = DataBindingUtil.inflate(inflater, R.layout.profile_info_fragment, container, false)
+        binding.lifecycleOwner = this
+
+        return binding.root
+    }
+
+    override fun onActivityCreated(savedInstanceState: Bundle?) {
+        super.onActivityCreated(savedInstanceState)
+        val activity = activity ?: return
+
+        viewModel = ViewModelProviders.of(activity).get(OpenChatInfoViewModel::class.java)
+        binding.viewModel = viewModel
+
+        setupViews()
+    }
+
+    private fun setupViews() {
+        setupToolbar()
+        setupProfileName()
+    }
+
+    private fun setupProfileName() {
+        displayNameEditText.addTextChangedListener(
+            TextUpdateWatcher(
+                ::updateProfileName,
+                OpenChatInfoViewModel.MAX_CHAT_NAME_LENGTH
```

Comment:
Currently, `OpenChatInfoViewModel.MAX_CHAT_NAME_LENGTH` is used for chatroom name and profile name. Even though the value is the same for them, but it actually represents different subjects. I think we need to declare an independent constant for them, separately.
```kotlin
const val MAX_CHAT_NAME_LENGTH = 50
const val MAX_PROFILE_NAME_LENGTH = 50
```

Reply:
changed. it should be 20 for profile name.